Ginger/Ginger/ValueExpression/ValueExpressionEditorPage.xaml.cs

```diff
@@ -993,38 +988,77 @@ private void UpdateHelpForCSFunction(object sender, RoutedEventArgs e)
                 }
 
             }
-            UpdateHelp(false, VER.Name, VER.Category, "Samples", samples, "Expression:" + System.Environment.NewLine + VER.Expression);
+            ShowSpecificHelp(VER.Name, VER.Category, "Samples", samples, "Expression:" + System.Environment.NewLine + VER.Expression);
+        }
+
+        private void ShowDefaultHelp()
+        {
+            xHelpPanelText.Text = string.Empty;
+            TextBlockHelper TBH = new TextBlockHelper(xHelpPanelText);
+
+            TBH.AddUnderLineText("Help:");
+            TBH.AddLineBreak();
+
+            string msg = "Enter expression like " + GingerDicser.GetTermResValue(eTermResKey.Variable) + " from different levels, Environment Parameter or function which will be calculated in run time into dynamic value."
+                           + Environment.NewLine + Environment.NewLine +
+                           "Multiple expressions can be added together."; ;
```

Comment:
Codacy found an issue: [Remove this empty statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6397283)